/polyx_events:
  polyx_node_speed:
    ros__parameters:
      use_sim_time: False
      duration:  1.0    #[sec]
      time: 5.0         #[sec]
      speed: 55.0       #[m/s]
      speed_rms: 88     #[mm/s]
      flags: 5

